### ISSUE TITLE
SQL parse errors, log full query

### ISSUE THREAD

**Issue by luigidellaquila** [orientechnologies/orientdb#5025]
As from the title, when OrientDB throws a parse exception it does not display full SQL statement. 

Having it in the logs will help to better debug applications

**Comment by ggoodman** [comment]
I think that this might have been triggered by my comments on gitter.

The relevant query is:

```sql
BEGIN
LET plunk = CREATE vertex plunk SET id = "0FdnGikEMeLkNKAc"
COMMIT
RETURN $plunk
```

and this produces the following...

Error thrown in Studio:

```
com.orientechnologies.orient.core.command.OCommandExecutorNotFoundException: Cannot find a command executor for the command request: sql.BEGIN LET plunk = CREATE vertex plunk SET id = "0FdnGikEMeLkNKAc" COMMIT RETURN $plunk
```

Trace in logs:

```
2015-09-28 07:22:43:670 FINE  [ONetworkProtocolHttpAbstract.execute] Requested: POST /command/plunker/sql/-/20 [ONetworkProtocolHttpDb]Caught exception
com.orientechnologies.orient.core.command.OCommandExecutorNotFoundException: Cannot find a command executor for the command request: sql.BEGIN LET plunk = CREATE vertex plunk SET id = "0FdnGikEMeLkNKAc" COMMIT RETURN $plunk
	at com.orientechnologies.orient.core.sql.OCommandExecutorSQLDelegate.parse(OCommandExecutorSQLDelegate.java:72)
	at com.orientechnologies.orient.core.sql.OCommandExecutorSQLDelegate.parse(OCommandExecutorSQLDelegate.java:42)
	at com.orientechnologies.orient.server.network.protocol.http.command.post.OServerCommandPostCommand.execute(OServerCommandPostCommand.java:76)
	at com.orientechnologies.orient.server.network.protocol.http.ONetworkProtocolHttpAbstract.service(ONetworkProtocolHttpAbstract.java:180)
	at com.orientechnologies.orient.server.network.protocol.http.ONetworkProtocolHttpAbstract.execute(ONetworkProtocolHttpAbstract.java:627)
	at com.orientechnologies.common.thread.OSoftThread.run(OSoftThread.java:77)

2015-09-28 07:22:43:699 SEVER Internal server error:
com.orientechnologies.orient.core.command.OCommandExecutorNotFoundException: Cannot find a command executor for the command request: sql.BEGIN LET plunk = CREATE vertex plunk SET id = "0FdnGikEMeLkNKAc" COMMIT RETURN $plunk [ONetworkProtocolHttpDb]
```

**Comment by luigidellaquila** [comment]
Hi @ggoodman 

not actually ;-) but it seems to be a common problem

Luigi

**Comment by lvca** [comment]
@ggoodman I think you don't use the script but you're executing it as simple command.

**Comment by luigidellaquila** [comment]
fixed in branch 2.1.x, porting it to develop

**Comment by luigidellaquila** [comment]
fixed in branches 2.1.x and develop.
Closing
